Add install info to GIS applications (#1723)
CSCfi/csc-user-guide: docs/apps/pcl.md

PCL is released under the terms of the BSD license, and thus free for commercial and research use.

## Citation

`Radu Bogdan Rusu and Steve Cousins, 2011: "3D is here: Point Cloud Library (PCL)" in IEEE International Conference on Robotics and Automation (ICRA)`

## Acknowledgement

Please acknowdlege CSC and Geoportti in your publications, it is important for project continuation and funding reports.
As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".

## Installation

> PCL was installed to Puhti via Singularity. The Singularity definition file is available on [CSCs 'singularity-recipes' repository](https://github.com/CSCfi/singularity-recipes/blob/main/pcl/pcl_from_ubuntu.def).

Reviewer: ```suggestion
PCL was installed to Puhti with Apptainer, using this [PCL Apptainer definition file](https://github.com/CSCfi/singularity-recipes/blob/main/pcl/pcl_from_ubuntu.def).
```
